admin/app/views/admin_main.scala.html

```diff
@@ -85,15 +85,15 @@
 
         <script type="text/javascript">
 
-            @Html(curlConfig().body)
-
             @Html(config(model.AdminPage(title)).body)
 
-            window.curl.paths['bootstraps/admin'] = '@Static("javascripts/bootstraps/admin.js")';
-
-            @Html(common.Assets.js.curl);
+            (function (document) {
```

Comment:
this could just be a script tag if we wanted, only admin. nbd though, no blocker.